addons/account_voucher/migrations/9.0.1.0/pre-migration.py

```diff
@@ -3,8 +3,15 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from openupgradelib import openupgrade
 
+column_copies = {
+    'account_voucher': [
+        ('voucher_type', None, None),
```

Comment:
Use 'type' instead of 'voucher_type', as this is how the field was named in v8. This will fix the current error in Travis.